.templates/postgres/service.yml

```diff
@@ -1,13 +1,13 @@
-postgres:
-  container_name: postgres
-  image: postgres
-  restart: unless-stopped
-  environment:
-    - POSTGRES_USER=postuser
-    - POSTGRES_PASSWORD=%randomPassword%
-    - POSTGRES_DB=postdb
-  ports:
-    - "5432:5432"
-  volumes:
-    - ./volumes/postgres/data:/var/lib/postgresql/data
+  postgres:
+    container_name: postgres
+    image: postgres
+    restart: unless-stopped
+    environment:
+      - POSTGRES_USER=${IOTSTACK_POSTGRES_USER:-postuser}
+      - POSTGRES_PASSWORD=${IOTSTACK_POSTGRES_INITIAL_PASSWORD:-IOtSt4ckpostgresDbPw} # Initial password when creating container, to change later see https://sensorsiot.github.io/IOTstack/Containers/PostgreSQL/
```

Comment:
We should probably alert the user if the password is left as the default one.

Reply:
Hmmm. How? The existing pattern of a placeholder which may or not be replaced by the menu doesn't "warn" anyone if the placeholder isn't substituted (best case the placeholder passes muster as a valid password; worst case the container won't come up). This Ukko extension is really just a variation on the theme. At least, that's my 2c on the topic.